quixstreams/sources/core/csv.py

```diff
@@ -1,74 +1,81 @@
 import csv
-import json
-from typing import Any, Callable, Optional
+import logging
+import time
+from pathlib import Path
+from typing import AnyStr, Callable, Optional, Union
 
 from quixstreams.models.topics import Topic
 from quixstreams.sources.base import Source
 
+logger = logging.getLogger(__name__)
+
 
 class CSVSource(Source):
     def __init__(
         self,
-        path: str,
-        dialect: str = "excel",
-        name: Optional[str] = None,
+        name: str,
```

Comment:
Why name required now?

Reply:
For consistency and flexibility.

There may be cases when you simply want to change the file name or stream data from a different file while keeping the same topic.
And picking the source name is relatively easy.

Also, file paths are not exactly unique (e.g., `./path.csv`  and `path.csv` can lead to the same file).
And if we use "path" as a name, the relative file paths cannot be used.